src/connectors/panoptic/panoptic.ts

```diff
@@ -318,18 +316,18 @@ export class Panoptic {
     longCallStrike: number, 
     straddleStrike: number, 
     asset: BigNumber,
-  ): Promise<BigNumber | unknown> {
+  ): Promise<any> {
     try {
       const panopticHelperAddress = this.PanopticHelper;
       const panopticHelperContract = new Contract(panopticHelperAddress, panopticHelperAbi.abi, wallet);
-      const tokenId = await panopticHelperContract.createBigLizard(
+      const response = await panopticHelperContract.createBigLizard(
```

Comment:
Yes, but this will depend on whether the method is state-changing or not. If state-changing, we need an extra line of logic to wait for the transaction to mine.

Reply:
First pass at simplification: 9d965853583346f11db3189de2d7ddd637e07610

Reply:
Done: 71e7f7cc0f74e12f791af82c4daa043f775f85da